README.md

```diff
@@ -105,33 +317,33 @@ username
 password
 ```
 
-# Step 3 :Prepare circuit to publish to Proof Market
+## Step 2: Prepare circuit to publish to Proof Market
 ```
 python3 scripts/prepare_statement.py -c=/root/tmp/zkllvm/build/examples/arithmetics_example.ll -o=arithmetic.json -n=arithmetic -t=placeholder-zkllvm
 ```
-
-# Step 4: See All Published circuits
+  
+## Step 4: See All Published circuits
 ```
 python3 scripts/statement_tools.py get
 ```
 
-# Step 5: Push a Bid
+## Step 5: Push a Bid
 ```
 python3 scripts/bid_tools.py push --cost <cost of the bid> --file <json file with public_input> --key <key of the statement> 
 ```
 
-# Step 6: Push an Ask
+## Step 6: Push an Ask
 ```
 python3 scripts/ask_tools.py push --cost <cost of the ask> --key <key of the statement> 
 ```
 
-# Step 7 : Fetch statements/inputs for proof generation
+## Step 7 : Fetch statements/inputs for proof generation
 ```
 python3 scripts/statement_tools.py get --key <key of the statement> -o <output file>
 python3 scripts/public_input_get.py --key <bid key> -o <output file path> 
```

Comment:
```suggestion
## Step 6: Fetch statements/inputs for proof generation
```bash
python3 scripts/statement_tools.py get --key <key of the statement> -o <output file>
python3 scripts/public_input_get.py --key <request key> -o <output file path> 
```